### PR TITLE
Linter: New - Add JQuery linter for deprecated functions (ED-1862)

### PR DESCRIPTION
**Pull request by StyleShit**
Added a `no-jquery` extension to ESlint to prevent JQuery deprecated functions.

Ref: https://www.npmjs.com/package/eslint-plugin-no-jquery

### CODE REVIEW

**Review by matipojo — DISMISSED** [review]

File: assets/dev/js/editor/elements/views/behaviors/sortable.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ SortableBehavior = Marionette.Behavior.extend( {
 
 		var $childViewContainer = this.getChildViewContainer(),
 			defaultSortableOptions = {
-				connectWith: $childViewContainer.selector,
+				connectWith: $childViewContainer,
```
Comment (kobizz) [inline assets/dev/js/editor/elements/views/behaviors/sortable.js:54]: Does it still work with other containers?

File: assets/dev/js/frontend/utils/lightbox/lightbox.js [hunk 2]
```diff
@@ -662,12 +662,12 @@ module.exports = elementorModules.ViewModule.extend( {
...
 					event.preventDefault();
 
-					$buttons.last().focus();
+					$buttons.last().trigger( 'focus' );
 				}
 			} else if ( isLast || ! focusedButton ) {
 				event.preventDefault();
 
-				$buttons.first().focus();
+				$buttons.first().trigger( 'focus' );
```
Comment (kobizz) [inline assets/dev/js/frontend/utils/lightbox/lightbox.js:670]: Make sure it still works like before (the previous behavior also focused the button)

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/elements/views/behaviors/sortable.js (hunk 1 above)
Comment (StyleShit) [reply assets/dev/js/editor/elements/views/behaviors/sortable.js:54]: Explanation?

Comment (kobizz) [reply assets/dev/js/editor/elements/views/behaviors/sortable.js:54]: The previous code affected other containers and allowed elements to be dragged to different areas (e.g. between sections). The new code seems to break this behavior.

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Pull request merged**